docker-compose.yml

```diff
@@ -13,6 +13,18 @@ services:
     environment:
       "PortabilityConfigurationServiceUrl": "http://portabilityservice.configurationservice"
 
+  portabilityservice.analysisengine:
+    image: ${DOCKER_REGISTRY}portabilityservice-analysisengine:${PORTABILITYSERVICE_VERSION:-latest}
+    build:
+      context: .
+      dockerfile: src/backend/PortabilityService.AnalysisEngine/Dockerfile
+    ports:
+      - "80:80"
```

Comment:
I think this should just be "80" (instead of "80:80") since we don't need to expose the analysis service externally (it will be called from the gateway).

Reply:
Fixed.  Yes I agree.